src/main/java/com/cdancy/bitbucket/rest/BitbucketApi.java

```diff
@@ -20,23 +20,24 @@
 import java.io.Closeable;
 
 import com.cdancy.bitbucket.rest.features.AdminApi;
-import com.cdancy.bitbucket.rest.features.BuildStatusApi;
 import com.cdancy.bitbucket.rest.features.BranchApi;
+import com.cdancy.bitbucket.rest.features.BuildStatusApi;
 import com.cdancy.bitbucket.rest.features.CommentsApi;
 import com.cdancy.bitbucket.rest.features.CommitsApi;
 import com.cdancy.bitbucket.rest.features.DefaultReviewersApi;
 import com.cdancy.bitbucket.rest.features.FileApi;
 import com.cdancy.bitbucket.rest.features.HookApi;
+import com.cdancy.bitbucket.rest.features.InsightsApi;
```

Comment:
:warning: Wrong lexicographical order for 'com.cdancy.bitbucket.rest.features.InsightsApi' import. Should be before 'java.io.Closeable'. (#QR85:30#)-CHECKSTYLE_CustomImportOrderCheck